### DISCUSSION TITLE
/posts/sketchybar-setup

### DISCUSSION THREAD

**Discussion by giscus** [josean-dev/sveltekit-blog#9] [General]
# /posts/sketchybar-setup

Use this guide along with my youtube video to make an amazing custom menu bar for your Mac!

https://www.josean.com/posts/sketchybar-setup

**Comment by noobie7** [comment]
Thanks for the blog. 

I have a few questions and a suggestion. 

### Suggestion : 
```Open ~/.config/sketchybar/items/media.sh and add the following code to it:```

This should be 
```Open ~/.config/sketchybar/plugins/media.sh and add the following code to it:```

### Question 
Can't this be a git repo and one could have cloned all the files directly? 
I mean apart from the installation steps?

**Reply by josean-dev** [reply]
Hey! Thanks for the feedback, really appreciate it. Will fix the issue ASAP and you can find the code here: [dev-environment-files](https://github.com/josean-dev/dev-environment-files?tab=readme-ov-file#sketchybar-custom-menu-bar-setup)

**Comment by Brodypen** [comment]
Howdy! One of the issues for me was that on Intel-macs homebrew is in the /usr/local instead of /opt.
So this line
```bash
mkdir -p ~/.config/sketchybar
cp /opt/homebrew/opt/sketchybar/share/sketchybar/examples/sketchybarrc ~/.config/sketchybar/sketchybarrc
mkdir ~/.config/sketchybar/plugins
cp -r /opt/homebrew/opt/sketchybar/share/sketchybar/examples/plugins/ ~/.config/sketchybar/plugins/
chmod +x ~/.config/sketchybar/plugins/*

```
will be 
```bash
mkdir -p ~/.config/sketchybar
cp /usr/local/opt/sketchybar/share/sketchybar/examples/sketchybarrc ~/.config/sketchybar/sketchybarrc
mkdir ~/.config/sketchybar/plugins
cp -r /usr/local/opt/sketchybar/share/sketchybar/examples/plugins/ ~/.config/sketchybar/plugins/
chmod +x ~/.config/sketchybar/plugins/*
```

Find more info [here](https://apple.stackexchange.com/questions/410825/apple-silicon-port-all-homebrew-packages-under-usr-local-opt-to-opt-homebrew)

**Comment by RomeuL** [comment]
Hello! Great content.

is it possible do make it so the skethybar is also visible on spaces where i have a fullscreen app?

**Comment by zedle** [comment]
Having an issue where the sf-symbols font is installed but i'm seeing :terminal: (placeholders) in the menu bar rather than the icons. Anyone know what the issue might be? Tried reinstalling the fonts and everything.

**Reply by zedle** [reply]
nevermind, got it, missed the sketchybar app font step

**Comment by bertrandmbanwi** [comment]
Nice work as always! If after doing this and after while i feel like removing it and doing something different, is the a step process in undoing this and getting rid of the spaces?

**Comment by mfrh89** [comment]
Very nice and superb tutorial! I love your work and style. 

Do you plan to embed Aerospace into Sketchybar to show its workspaces? As fas as I‘m aware you ditched Yabai, right?

**Comment by rafay99-epic** [comment]
I took the basic config and I started with your blog post and more over the video and now I am modifying to fit my need, thanks for the blog and video, becase installing is the pain.
